Apps/WebClient/src/ClientApp/public/index.html

```diff
@@ -62,5 +61,8 @@
     </head>
     <body class="d-flex flex-column">
         <div id="app-root">Loading...</div>
+        <div id="unsupported-browser"></div>
+        <script src="/modernizr-custom.js"></script>
+        <script src="/index.js"></script>
```

Comment:
intead of index.js lets name it detectBrowser or something like that.

Reply:
also, does the modernizer script need to be in the body?

Reply:
I believe the script tags are recommended to be at the end of the html file. Per the following screenshot, VueJS generates their scripts and inserts them into the same location:
![image](https://user-images.githubusercontent.com/48332333/93539099-a44e4600-f904-11ea-8ed7-1ff0d4a6587e.png)

Reply:
and per https://www.w3schools.com/js/js_whereto.asp , "Placing scripts at the bottom of the <body> element improves the display speed, because script interpretation slows down the display."